AstSemantics.md

```diff
@@ -426,9 +426,8 @@ The same operators are available on 64-bit integers as the those available for
 ## Floating point operators
 
 Floating point arithmetic follows the IEEE 754-2008 standard, except that:
- - The sign bit and fraction field of any NaN value returned from a floating
-   point arithmetic operator are deterministic under more circumstances than
-   required by IEEE 754-2008.
+ - The IEEE 754-2008 recommendation that operations propagate NaN bits from
+   their operands is permitted but not required.
```

Comment:
It's an area where wasm doesn't "follow" IEEE's recommendation.

I added a reference to the relevant section.